<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Mapa</string>
    </property>
    <widget class="QPushButton" name="openImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wczytaj obraz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Zapisz obraz</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomIn">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomOut">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zoomNo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Algorytm</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeszukiwanie w głąb</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Przeszukiwanie wszerz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="symSpeed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Prędkość symulacji</string>
    </property>
    <widget class="QSlider" name="symSpeedVal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="value">
      <number>-10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="symSpeedText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>operacji/sekundę: 0.1</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="symColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Obszar kolorowania</string>
    </property>
    <widget class="QSlider" name="symColorVal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>40</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="symColorText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>zakres kolorowania: 100</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
